test/runnable/testscope2.d

```diff
@@ -29,7 +29,7 @@ pragma(msg, "foo4 ", typeof(&SS.foo4));
 
 void test3()
 {
-    version (all)
+    version (none)
     {
         import std.stdio;
         writeln(SS.foo1.mangleof);
```

Comment:
You could change the following checks to `static assert`s to make these `writeln`s obsolete.

Reply:
Expand the diff below, it's already done.

Reply:
There are only runtime `assert`s below?

Reply:
Yes, run-time or compile-time makes no difference for checking the mangle. The writelns are only useful for debugging in the event that the asserts fail.

Reply:
A `static assert` yields better error  messages in the CI logs.

```"<Actual mangling>" == "<Expected mangling>" is false```

instead of 

```<File>(<Line>): Assertion failure```

(Anyway, the current change is fine and  we can always change that test if it becomes cumbersome to work with)